roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -1,7 +1,14 @@
 # These tasks are run when contiv_network_mode is set to aci
 
-- name: pull aci-gw container
-  shell: docker pull contiv/aci-gw
+# Load the aci-gw-image from local tar. Ignore any errors to handle the
+# case where the image is not built in
+- name: copy aci-gw image
+  copy: src={{ contiv_standalone_binaries }}/aci-gw-image.tar dest=/tmp/aci-gw-image.tar
+  ignore_errors: yes
```

Comment:
Is there a condition for source file exists? I was only able to find stat for destination file? Other option is to have it passed in an a parameter to the installation.